# DHI patch for the NuGet.config of azure-functions-extension-bundles.
# Restores from nuget.org instead of the Azure DevOps upstream-public feed.
# Upstream: https://github.com/Azure/azure-functions-extension-bundles
diff --git a/NuGet.config b/NuGet.config
--- a/NuGet.config
+++ b/NuGet.config
@@ -2,10 +2,10 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="upstream-public" value="https://pkgs.dev.azure.com/azfunc/public/_packaging/upstream-public/nuget/v3/index.json" protocolVersion="3" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
   </packageSources>
   <packageSourceMapping>
-    <packageSource key="upstream-public">
+    <packageSource key="nuget.org">
       <package pattern="*" />
     </packageSource>
   </packageSourceMapping>
